src/index.njk

```diff
@@ -145,11 +108,9 @@ tags: index-page # Note: This adds a custom class to layout.njk
         <div class="icon-box">
           <div class="icon"><i class="bx bx-sort-down"></i></div>
           <h4><a href="https://github.com/Kuadrant/limitador">Limitador</a></h4>
-          <p>Limitador is a generic rate-limiter written in Rust. It can be used as a library, or as a service. The service exposes HTTP endpoints to apply and observe limits. Limitador can be used with Envoy because it also exposes a grpc service, on a different port, that implements the Envoy Rate Limit protocol (v3).</p>
+          <p>Limitador is a generic rate-limiter written in Rust. Limits are defined by Kuadrant's RateLimitPolicy and enforced by limitador.  It can be used as a library, or as a service. The service exposes HTTP endpoints to apply and observe limits. Limitador can be used with Envoy because it also exposes a grpc service, on a different port, that implements the Envoy Rate Limit protocol (v3).</p>
```

Comment:
```suggestion
          <p>Limitador is a generic rate-limiter written in Rust. Limits are defined by Kuadrant's RateLimitPolicy and enforced by Lmitador.  Limitador can be used as a library, or as a service. The service exposes HTTP endpoints to apply and observe limits. Limitador can be used with Envoy because it also exposes a gRPC service, on a different port, that implements the Envoy Rate Limit protocol (v3).</p>
```

Reply:
> Limitador can be used with Envoy because it also exposes a gRPC service

To be fair, this is also true about Authorino.